docs/development/howtocontribute.md

```diff
@@ -5,52 +5,49 @@ description: "How to contribute"
 group: development
 ---
 
-## IMPORTANT
+# Contributing to Apache Zeppelin ( Code )
 
+## IMPORTANT
 Apache Zeppelin (incubating) is an [Apache2 License](http://www.apache.org/licenses/LICENSE-2.0.html) Software.
-Any contribution to Zeppelin (Source code, Documents, Image, Website) means you agree license all your contributions as Apache2 License.
-
+Any contributions to Zeppelin (Source code, Documents, Image, Website) means you agree with license all your contributions as Apache2 License.
 
+## Setting up
+Here are some tools you will need to build and test Zeppelin. 
 
-### Setting up
-Here are some things you will need to do to build and test Zeppelin. 
+#### Software Configuration Management ( SCM )
 
-#### Software Configuration Management(SCM)
+Since Zeppelin uses Git for it's SCM system, you need git client installed in your development machine. 
 
-Zeppelin uses Git for it's SCM system. Hosted by github.com. `https://github.com/apache/incubator-zeppelin` You'll need git client installed in your development machine. 
-
-#### Integrated Development Environment(IDE)
+#### Integrated Development Environment ( IDE )
 
 You are free to use whatever IDE you prefer, or your favorite command line editor. 
 
-#### Build Tools
+### Build Tools
 
 To build the code, install
-Oracle Java 7
-Apache Maven
+  
+  * Oracle Java 7
+  * Apache Maven
 
-### Getting the source code
-First of all, you need the Zeppelin source code. The official location for Zeppelin is [https://github.com/apache/incubator-zeppelin](https://github.com/apache/incubator-zeppelin)
+## Getting the source code
+First of all, you need Zeppelin source code. The official location of Zeppelin is [http://git.apache.org/incubator.git](http://git.apache.org/incubator-zeppelin.git).
```

Comment:
`[http://git.apache.org/incubator.git](http://git.apache.org/incubator-zeppelin.git)`
I think this should be
`[http://git.apache.org/incubator-zeppelin.git](http://git.apache.org/incubator-zeppelin.git)`